Packs/PrismaCloud/Integrations/RedLock/RedLock.yml

```diff
@@ -54,6 +54,22 @@ script:
   type: python
   subtype: python2
   commands:
+  - name: redlock-get-RQL-response
+    description: Run RQL query on Prisma Cloud
+    arguments:
+    - name: rql  
+      required: true
+      default: false
+      isArray: false
+      description: the RQL query to run
```

Comment:
add explanation/link on how to write an RQL query

Reply:
added link to examples